orttraining/orttraining/python/training/_ortmodule_utils.py

```diff
@@ -13,17 +13,21 @@
 from torch.utils.cpp_extension import load_inline
 
 
-def _ortvalue_to_torch_tensor(ortvalue):
+def _ortvalue_to_torch_tensor(ortvalue, inner = False):
```

Comment:
Add a docstring to explain what `inner` means here.

Reply:
Would it make sense to make it always behave as if `inner=True`, remove the arugment, and have the caller deal with whether or not to unwrap/wrap a layer of `OrtValue` object? (For this method and the one below)

Reply:
That is a good suggestion to simplify the code but can we please keep default to false so that training workload does not incur extra cycles to wrap/unwrap?

---
In reply to: [613578263](https://github.com/microsoft/onnxruntime/pull/7324#discussion_r613578263) [](ancestors = 613578263)

Reply:
Oh yes, I would go further and make it *only* deal with the no-wrapping case: i.e. remove the argument, and push the wrapping to the existing callsites. Then you don't even need to pay for the extra arg and branch in the training case :).